src/main/java/com/thedariusz/warnme/api/MeteoAlertController.java

```diff
@@ -1,27 +1,85 @@
 package com.thedariusz.warnme.api;
 
 import com.thedariusz.warnme.twitter.TweetService;
+import com.thedariusz.warnme.user.User;
+import com.thedariusz.warnme.user.repository.UserDao;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.authentication.AnonymousAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.bind.annotation.RestController;
 
-@RestController
+//@RestController
+@Controller
 @RequestMapping(value = "/alerts")
 public class MeteoAlertController {
 
     private final TweetService tweetService;
+    private final UserDao dao;
 
     @Autowired
-    public MeteoAlertController(TweetService tweetService) {
+    public MeteoAlertController(TweetService tweetService, UserDao dao) {
         this.tweetService = tweetService;
+        this.dao = dao;
     }
 
     @PostMapping("/{id}")
+    @ResponseBody
```

Comment:
what is the purpose of this annotation here if you simply do not return anyhing ?